docs/src/tutorial/strings.rst

```diff
@@ -322,23 +315,17 @@ you can use the following approach.
 In general, for arguments of external C functions, the ``const``
 modifier does not matter and can be left out in the Cython
 declaration (e.g. in a .pxd file).  The C compiler will still do
-the right thing, even if you declare this to Cython::
+the right thing, even if you declare this to Cython:
 
-    cdef extern from "someheader.h":
-        int process_string(char* s)   # note: looses API information!
+.. literalinclude:: ../../examples/tutorial/string/const_left_out.pyx
 
 However, in most other situations, such as for return values and
 variables that use specifically typedef-ed API types, it does matter
 and the C compiler will emit at least a warning if used incorrectly.
 To help with this, you can use the type definitions in the
-``libc.string`` module, e.g.::
-
-    from libc.string cimport const_char, const_uchar
+``libc.string`` module, e.g.:
```

Comment:
Now that Cython has direct support for the ``const`` modifier, this ugly work-around should be removed from the docs and it should simply show how to declare and use APIs that use ``const``.